### ISSUE TITLE
No resource identifier found for attribute 'ms_enableFloatingLabel'

### ISSUE THREAD

**Issue by raquezha** [ganfra/MaterialSpinner#40]
Hello can you help me resolve this?

<img width="737" alt="by default 2015-11-09 at 2 09 15 pm" src="https://cloud.githubusercontent.com/assets/9584016/11026965/844f8e94-86eb-11e5-89a9-314bc16eac90.png">

in XML:
```
 <fr.ganfra.materialspinner.MaterialSpinner
                android:id="@+id/spGender"
                android:layout_width="fill_parent"
                android:layout_height="wrap_content"
                android:entries="@array/gender_array"
                app:ms_enableFloatingLabel="false"
                app:ms_multiline="false"
                app:ms_hint="Gender"
                app:ms_floatingLabelText="Gender"
                app:ms_alignLabels="true"/>
```

Gradle:
```
dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    ...
    compile ('com.github.ganfra:material-spinner:1.1.0')
    ...
}
```

I'm Using Android Studio 1.5
 compileSdkVersion 23
 buildToolsVersion '23.0.2'

**Comment by akamud** [comment]
From what I can see, this is not available in version 1.1.0. It is a new feature that has not been released to Gradle yet.

[This is the commit](https://github.com/ganfra/MaterialSpinner/commit/e7419a265b6024c52a31998dfb1e3bc17af374fd) related to this new feature.

**Comment by ganfra** [comment]
Yes, sorry about that. The next release will be soon available
